src/aura/BGE_BatchGiftEntryTab/BGE_BatchGiftEntryTab.cmp

```diff
@@ -81,6 +82,19 @@
         </div>
     </header>
 
+    <aura:if isTrue="{!v.showRetirementPrompt}">
+        <c:utilPrompt 
+            aura:id="retirementPrompt" 
+            id="retirementPrompt"
+            headerTitle="{!$Label.c.bgeRetirementWarningHeader}"
+            promptMessage="{!$Label.c.bgeRetirementWarningBody}"
+            confirmButtonTitle="Confirm" 
```

Comment:
Took a quick look and I'm surprised we don't have a standalone common label for 'Confirm'. I think 'Confirm' is worthy as a common label `commonConfirm`.

Custom labels we prefix with 'common' are expected to used across NPSP and are not specific to any given feature. What do you think?

@melissabarber - additional thoughts?